content/scripts/constants.js

```diff
@@ -8,16 +8,18 @@ require("dotenv").config();
 // const DEFAULT_DESTINATION =
 //   process.env.ROOT || path.join(__dirname, "..", "..", "client", "build");
 
+const BUILD_TIMEOUT = process.env.BUILD_TIMEOUT || 1800;
+
 const DEFAULT_DATABASE_URL =
   process.env.DATABASE_URL || "mysql2://username:password@host/databasename";
 
 const DEFAULT_BUILD_LOCALES = (process.env.BUILD_LOCALES || "")
   .split(",")
-  .filter((x) => x);
+  .filter(x => x);
```

Comment:
I'm so confused why this is happening!!! I verified that I'm using `prettier` 2.0.2 when I'm doing the `npm run prettier-format` command.